datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
If to be done through save I guess I need to replace my commit method in that Annexificator with save but actually not sure atm if I do commit while crawling at the superdataset level since usually I want to review before committing... So I guess it would just need an explicit add after create to mimic previous behavior